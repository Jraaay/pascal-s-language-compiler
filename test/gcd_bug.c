#include<stdio.h>

/* Error,  Parser gives no AST. */