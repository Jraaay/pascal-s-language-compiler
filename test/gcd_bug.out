{
    "ast": null,
    "symbolTable": {
        "constants": [],
        "variables": [],
        "subFunc": []
    },
    "error": [
        {
            "code": "B-01",
            "info": {
                "line": 7,
                "value": [
                    "="
                ],
                "lexpos": 121
            }
        }
    ],
    "warning": []
}